pscheduler-test-throughput/throughput/validate.py

```diff
@@ -52,11 +52,15 @@
             "additionalProperties": False,
             "properties": {
                 "throughput-bits": {
-                    "description": "Summarized view of the overall throughput rate in bits/second",
+                    "description": "Summarized view of the overall sender throughput rate in bits/second",
                     "type": "number"
                     },
                 "throughput-bytes": {
-                    "description": "Summarized view of the overall throughput rate in bytes/second",
+                    "description": "Summarized view of the overall sender throughput rate in bytes/second",
+                    "type": "number"
+                    },
+                "receiver-throughput-bits": {
```

Comment:
In theory, this should require a bump in the schema version and alterations to the validator to properly validate one or the other.  There isn't anything outside the test plugin that depends on it and older software will simply not know it was there and ignore it.